pkg/scheduler/algorithm/priorities/image_locality_test.go

```diff
@@ -108,6 +108,12 @@ func TestImageLocalityPriority(t *testing.T) {
 		},
 	}
 
+	node00000 := v1.NodeStatus{}
+
+	node00001 := v1.NodeStatus{
+		Images: []v1.ContainerImage{},
```

Comment:
It seems the 2 status work the same, right? If so, I'm leaning towards renaming to `nodeWithNoImage`.